draft-ietf-rats-corim.md

```diff
@@ -1254,11 +1254,28 @@ applies to all measurements in the triple, including those in `measurement-value
 # CoBOM {#sec-cobom}
 
 A Concise Bill of Material (CoBOM) object represents the signal for the
-Verifier to activate the listed tags. Data contained in a tag MUST NOT be used
+Verifier to activate the listed tags. Data contained in a tag SHOULD NOT be used
 for appraisal until a CoBOM which activates that tag has been received and
 successfully processed. All the tags listed in the CoBOM must be activated in
 the same transaction, i.e., either all or none.
 
+The number of CoBOMs required in a given supply chain eco-system is dependent on
+Verifier Owner's Appraisal Policy for Evidence, which is often driven by the complexity and nature of
+the use case.
+
+For example, in cases where only limited CoRIMs are produced by a single authority, a Verifier Owner may have
+a policy that CoBOM's are not required. In this case, signature chain over each CoRIM is sufficient authorization,
+and may not require additional authorization via CoBOM. Any tags in a CoRIM received by a Verifier
+are activated immediately and treated valid for appraisal
+
+However, a more common case is the one where Verifier receives multiple CoRIMs containing
+Reference Values and Endorsements from multiple Reference Value providers and Endorsers
+respectively. In such cases, an integrator or a supplier authority may be designated to
```

Comment:
It's not clear what a _"supplier authority"_ is.  I reckon "integrator" is enough to convey the idea.

```suggestion
Reference Values and Endorsements from multiple Reference Value providers and Endorsers.  An integrator may be designated to
```

Reply:
> It's not clear what a _"supplier authority"_ is. I reckon "integrator" is enough to convey the idea.

I have no hard views on this, I added supplier authority as per Ned's suggestion. Keeping supplier authority does not hurt, as it may not necessarily be the integrator but can also be an authority

Reply:
> Keeping supplier authority does not hurt as it may not necessarily be the integrator but can also be an authority

I guess I'm having a hard time to digest the "authority" bit, because the integrator *is* an "authority" too given it signs the CoRIM...

Maybe: _"[...] In such cases, an integrator or a specific supplier may be designated to [...]"_

Reply:
> > Keeping supplier authority does not hurt as it may not necessarily be the integrator but can also be an authority
> 
> I guess I'm having a hard time to digest the "authority" bit, because the integrator _is_ an "authority" too given it signs the CoRIM...
> 
> Maybe: _"[...] In such cases, an integrator or a specific supplier may be designated to [...]"_

Authority (in this document) refers to the key used sign (assert) conceptual messages or the key designated by an appraisal policy as the entity authorized to make assertions about a class of claims. The former case is relevant when no policy exists, but is overridden if a policy does exist.

Reply:
Thinking a bit more, perhaps we can retain supplier authority and make it like below:

 In such cases, a supplier authority (for example, an integrator) may be designated to